### PR TITLE
Design maintenance - November 21

### PR DESCRIPTION
**Pull request by micheal-parks**
One simple thing here: align badges with our semantic names!

When this is merged I'll open PRs downstream updating all our badges.

### CODE REVIEW

File: packages/core/src/lib/icon/icons.ts [hunk 1]
```diff
@@ -42,7 +42,6 @@ export const paths = {
   'content-duplicate': MDI.mdiContentDuplicate,
   'content-save-outline': MDI.mdiContentSaveOutline,
   'credit-card-outline': MDI.mdiCreditCardOutline,
-  'database-outline': MDI.mdiDatabaseOutline,
```
Comment (micheal-parks) [inline packages/core/src/lib/icon/icons.ts:45]: Oh also nat wanted this removed ✂️

File: packages/core/src/lib/__tests__/badge.spec.ts [hunk 2]
```diff
@@ -5,72 +5,75 @@ import { cxTestArguments, cxTestResults } from './cx-test';
...
-    render(Badge, { variant: 'gray', label: 'Inactive' });
+    render(Badge, { variant: 'inactive', label: 'Inactive' });
     expect(screen.getByText('Inactive')).toBeInTheDocument();
   });
 
-  it('Renders a gray badge if the variant is specified as gray', () => {
-    render(Badge, { variant: 'gray', label: 'Inactive' });
-    const grayBadge = screen.getByText('Inactive');
-    expect(grayBadge).toHaveClass(
+  it('Renders a inactive badge if the variant is specified as inactive', () => {
```
Comment (mcous) [inline packages/core/src/lib/__tests__/badge.spec.ts:12]:
```suggestion
  it('Renders an inactive badge if the variant is specified as inactive', () => {
```

😜

File: packages/core/src/lib/__tests__/badge.spec.ts [hunk 3]
```diff
@@ -5,72 +5,75 @@ import { cxTestArguments, cxTestResults } from './cx-test';
...
       'bg-success-light',
       'text-success-dark'
     );
   });
 
-  it('Renders an orange badge if the variant is specified as orange', () => {
-    render(Badge, { variant: 'orange', label: 'Danger' });
-    const orangeBadge = screen.getByText('Danger');
-    expect(orangeBadge).toHaveClass(
+  it('Renders an warning badge if the variant is specified as warning', () => {
```
Comment (mcous) [inline packages/core/src/lib/__tests__/badge.spec.ts:42]:
```suggestion
  it('Renders a warning badge if the variant is specified as warning', () => {
```

**Review by mcous — APPROVED** [review]
✨

**Review by anjinai — APPROVED** [review]
🎉

**Pull request merged**